css/main.css

```diff
@@ -1,3 +1,663 @@
+  margin: 1rem auto;
+}
+
+/* main / main-article */
+
+.main-article {
+  background-color: var(--light-green);
+}
+
+.main-article img {
+  width: 280px;
+  height: 377px;
+  object-fit: cover;
+  object-position: 80% 50%;
+  margin: 1.5rem;
+}
+
+ .article-btn {
+  background-color: transparent;
+  color: var(--black);
+  width: 160px;
+  height: 48px;
+  margin-top: 10px;
+  margin-bottom: 25px; 
+}
+
+/* main / features-article-1 */
+
+.features-article-1 {
+  background-color: var(--lylac);
+  padding: 1.2rem 0;
+}
+
+.features-article-1 img {
+  width: 280px;
+  height: 274px;
+  object-fit: cover;
+  object-position: 50% 50%;
+  margin: 1.5rem 0;
+  
+}
+
+/* main / features-article-2 */
+
+.features-article-2 img {
+  width: 280px;
+  height: 371px;
+  object-fit: cover;
+  object-position: 50% 50%;
+  margin-top: 25px;
+  margin-bottom: 30px;
+}
+
+/* article/ delivery-article */
+
+.delivery-options {
+  background-color: var(--light-green);
+  margin: auto;
+  text-align: center;
+}
+
+.delivery-options .option {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  height: 200px;
+}
+
+/* footer */
+
+footer {
+  background-color: var(--dark-green);
+  color: var(--white);
+  text-align: left;
+}
+
+footer h1 {
+  color:  var(--black);
+  text-align: center;
+}
+
+.follow-us {
+  background-color: var(--white);
+  height: 180px;
+  padding-top: 2rem;
+}
+
+footer img {
+  position: relative;
+  width: 131px;
+  height: 159px;
+  object-fit: cover;
+  object-position: 50% 50%;
+}
+
+.image-container {
+  height: 250px;
+
+}
+
+.img-1 {
+  transform: rotate(-20deg);  
+  left: 75px;
+  top: 70px;
+}
+
+.img-2 {
+  transform: rotate(20deg);
+  right: 100px;
+  bottom: 20px;
+}
+
+.img-3 {
+  transform: rotate(-20deg);
+  left: 90px;
+  bottom: 270px;
+  }
+
+.img-4 {
+  transform: rotate(20deg);
+  left: 10px;
+  bottom: 170px;
+  }
+
+.footer-nav {
+  display: flex;
+  justify-content: space-between;
+  padding: 25px;  
+}
+
+ul, label{
+  font-family:"HelveticaNeueW01-45Ligh";
+  list-style-type:none;
+  padding-left: 0;
+  line-height: 2;
+  font-weight: 50;  
+}
+
+.footer-email {
+  padding: 1.5rem;
+}
+
+.email-submit {
+  display: flex;
+  flex-direction: row;
+  align-items: end;
+  margin: 2rem 0;
+  bottom: 0;
+}
+
+.submit-btn {
+  background-color: transparent;
+  border:2px solid var(--white);
+  width: 160px;  
+}
+
+.email-submit input {
+  width: 120px;
+  bottom: 0;
+  border: none;
+  border-bottom: 2px solid var(--white) ;
+  background-color: transparent;
+}
+
+.juice-bar-logo img{
+  width: 80px;
+  height: auto;
+}
+
+.footer-bottom {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  padding: 0 1.3rem;
+  height: 200px;
+}
+
+.footer-bottom .footer-note{
+  line-height: 1.1;
+  font-size: 14px;
+  margin: 0;
+  text-align: left;
+}
+
+a {
+  color: var(--white);
+}
+
+/* breakpoint 2 --> 900px */
+
+@media screen and (min-width: 900px) {
```

Comment:
well done for styling this mobile first. It's best to have more than one breakpoint to account for desktop and tablet